util/SerializeOrderSet.cpp

```diff
@@ -34,8 +36,15 @@ BOOST_CLASS_EXPORT(ForgetOrder)
 template <class Archive>
 void Order::serialize(Archive& ar, const unsigned int version)
 {
-    ar  & BOOST_SERIALIZATION_NVP(m_empire)
-        & BOOST_SERIALIZATION_NVP(m_executed);
+    ar  & BOOST_SERIALIZATION_NVP(m_empire);
+    /* m_executed is intentionally not serialized so that an order is run once
```

Comment:
discuss how this works when loading a game? ie. all orders re-executed when a game is loaded, assuming the saved gamestate is that before any orders were executed by the client?